.idea/checkstyle-idea.xml

```diff
@@ -20,4 +20,4 @@
       </map>
     </option>
   </component>
-</project>
+</project>
```

Comment:
Could you unstage these changes to these `.idea` files to keep the changes in this PR clean? It's possible we need to regenerate some of these `.idea` files, though.

Reply:
That's weird, I don't recall ever touching this file. Was it auto-generated?

Reply:
Yeah, these files are IntelliJ IDEA configuration files, and the IDE can edit this file if certain things change while you're editing the files. It's a little annoying that this can happen, but the upside is that the config files are all "ready to go" when someone checks out the repo. The solution is to check in those files periodically and otherwise to not commit changes to them. There _might_ be some solution to make that user friendly--I just always use `git` from the command line and so only check in the relevant directories (rather than all changes).